documentation/smg/14_documentation.md

```diff
@@ -0,0 +1,45 @@
+# Documentation
+
+This document will outline the process surrounding the documentation currently housed in the `horace-docs` repository. It assumes a basic knowledge of Restructured Text (ReST) format (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html) especially the subset of the sphinx-doc toolset (https://www.sphinx-doc.org/en/master/)
+
+## Documentation style guide
+
+
+## Adding new documents
+### Create new `.rst`
+
+The first job to add a new document is to create it. 
+
+* The file should be added into the `docs` folder, preferably with the `.rst` suffix. 
+   * For ease the filename of the document should be the title with spaces replaced by underscores. 
```

Comment:
Presuming that the ease is something like "for ease of reference ... " or something; would make it clearer why we do this.

Reply:
Updated